### DISCUSSION TITLE
mitm does not work anymore starting with Android Chrome v98

### DISCUSSION THREAD

**Discussion by VorticonCmdr** [mitmproxy/mitmproxy#5199] [Q&A]
Hi,

I had a working setup on my rooted Pixel 4 Android 12 device which allowed me to proxy all traffic in Chrome. Starting a while ago (I believe in version 98 of Chrome) it doesn't anymore. The setup itself does not seem to be the problem because other apps work just fine. The (advanced) error message Chrome shows the user implies there seems to be an HSTS issue but I believe (not sure) it is not only limited to domains using HSTS (not sure).

An example error mitmproxy reports is:

`Warn: 192.168.x.x:xxxxx: Client TLS handshake failed. The client does not trust the proxy's certificate for clients4.google.com (OpenSSL Error([('SSL routines', 'ssl3_read_bytes', …(more in eventlog)`

Can anyone confirm this or has any ideas on how to solve this issue?

Cheers
Valentin

**Comment by mhils** [comment]
Do you see this happening for non-Google domains as well? It might very well be the case that there's additional certificate pinning for the Google domains.

**Reply by VorticonCmdr** [reply]
Yes. I just double-checked with two domains I know (and re-checked) who don't have HSTS. While at it I checked with Android Chrome Beta & Dev as well.

I also checked with different apps. One being Upday (which worked just fine) and one being Twitter (which had the very same issue). So there might be something deeper (OS-level) involved?

I don't want to rule out that this is a local issue but it sure doesn't look like it to me. It would be great if someone with an up-to-date Android 12 and Chrome could verify (or falsify) this.

**Reply by VorticonCmdr** [reply]
Oh and in case it is important. I don't know if it used to work with Upday and Twitter before. I only can say this for sure for Chrome.
